### PR TITLE
Small n easy fixes

### PR DESCRIPTION
**Pull request by ttomsu**
Bunch of quick fixes for rules that only have a few instances

### CODE REVIEW

File: spring-cloud-gcp-data-datastore/src/main/java/com/google/cloud/spring/data/datastore/core/LazyUtil.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
  */
 final class LazyUtil {
 
-	static private final ObjenesisStd objenesis = new ObjenesisStd();
+	static final private ObjenesisStd objenesis = new ObjenesisStd();
```
Comment (dzou) [inline spring-cloud-gcp-data-datastore/src/main/java/com/google/cloud/spring/data/datastore/core/LazyUtil.java:44]: This one should be `private static final`

File: spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/CommitTimestamp.java [hunk 2]
```diff
@@ -38,7 +38,7 @@
  * @since 1.2.2
  */
 public final class CommitTimestamp {
-	private static final Map<Object, Object> VALUES = new ConcurrentHashMap<>();
+	private final static Map<Object, Object> VALUES = new ConcurrentHashMap<>();
```
Comment (dzou) [inline spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/CommitTimestamp.java:41]: This one too should still be `private static final`?

**Review by dzou — APPROVED** [review]

File: spring-cloud-gcp-data-datastore/src/main/java/com/google/cloud/spring/data/datastore/core/LazyUtil.java (hunk 1 above)
Comment (ttomsu) [reply spring-cloud-gcp-data-datastore/src/main/java/com/google/cloud/spring/data/datastore/core/LazyUtil.java]: Whoops - fixed.

File: spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/CommitTimestamp.java (hunk 2 above)
Comment (ttomsu) [reply spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/CommitTimestamp.java]: Whoops - fixed.

**Pull request merged**